arrays/arrays.c

```diff
@@ -35,9 +38,9 @@ Array *create_array (int capacity) {
 void destroy_array(Array *arr) {
 
   // Free all elements
-
+  free(arr -> elements);
```

Comment:
Since you opted to use strdup() to add elements to the array (which is the ideal course of action) you have to also free each individual element in the elements array. strdup() does a malloc() call under the hood so there's a bit more allocated memory that needs to be accounted for when you destroy an array.